Jenkinsfile

```diff
@@ -0,0 +1,87 @@
+pipeline {
+    agent {label 'mo1833.gadi'}
+    environment {
+        DB_PATH = '/g/data/ik11/databases'
+        DB_LINK = "${DB_PATH}/cosima_master.db"
+        DB_DATE = """${sh(returnStdout: true, script: 'date +"%Y-%m-%d"')}""".trim()
+        DB      = "${DB_PATH}/daily/cosima_master_${DB_DATE}.db"
+    }
+    stages {
+        stage('Update database') {
+            steps {
+                // Check if an update of the database has already been done today. Stop with an error if that is the case.
+                script {
+                    if (fileExists("${DB}")) {
+                        error("Database has already been updated today.")
+                    }
+                }
+
+                // Start the update from the latest version of the database
+                sh '''cp ${DB_LINK} ${DB}
+                      chmod 640 ${DB}'''
+
+                // Try to update the database. Because the PBS job has a time limit, the update can fail for two different reasons:
+                //  1. the time limit was reached
+                //  2. an error occured
+                // In the first case, we don't want to mark the build as failed, because the database will still be usable.
```

Comment:
I assumed it would be better, as it contains more indexed files, but here I'll follow what you think is best, as you know better how people use the database. My only concern would be a situation where there are lots of new files added on a daily bases. Then it might take many days until the "live" database is updated.

Reply:
OK, I see your point, let's go with your version. I guess it means that a really huge DB update (e.g. cycle 4) might take a few days to be completed, so the new data would only be partially accessible until then.